examples/Atlas/+atlasControllers/AtlasPlanEval.m

```diff
@@ -2,7 +2,7 @@
 % A PlanEval which includes some Atlas-specific tweaks. Specifically, it
 % provides a cache of useful robot properties which may be expensive to look
 % up at runtime.
-  properties (Access = private)
+  properties (Access = protected)
```

Comment:
Sorry, I said modify, but should have just said access. The DRCplaneval just needed read access to it